changelog/buildpacks/_posts/2020-03-25-php-7.4.4.markdown

```diff
@@ -0,0 +1,11 @@
+---
+modified_at: 2020-03-25 10:00:00
+title: 'PHP - Support of version 7.4.4 and 7.2.29, 7.3.16'
+github: 'https://github.com/Scalingo/php-buildpack'
+---
+
+Changelogs:
+
+* [https://www.php.net/ChangeLog-7.php#PHP_7_4](https://www.php.net/ChangeLog-7.php#7.4)
```

Comment:
```suggestion
* [https://www.php.net/ChangeLog-7.php#7.4.4](https://www.php.net/ChangeLog-7.php#7.4.4)
```

Can you update all changelog links to target the precise version?

Reply:
Aaah, it is a dot